### ISSUE TITLE
Assign accessible labels to UI elements

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rutherford-scattering#135]
This sim is likely the next to be deployed with keyboard navigation. @emily-phet @terracoda can you please prioritize/assign working on labels for this sim?

**Comment by terracoda** [comment]
@jessegreenberg, ok will do.

**Comment by emily-phet** [comment]
@ariel-phet Let's use this issue to track progress with the Rutherford Scattering summary/labels/help text. 
So far - I drafted an initial structure with placeholders.
@ariel-phet drafted initial content.

@terracoda - let's discuss this briefly this week to give some feedback. 
Goal is to iterate and have this complete by Feb. 15th.

**Comment by terracoda** [comment]
Tuesday this week, @ariel-phet and I met to discuss changes I made to the initial draft. We think we have a solid draft for screen 1. @ariel-phet agreed to apply what we did in screen 1 to screen 2.

**Comment by terracoda** [comment]
@emily-phet, one thing currently missing from screen 1 is a description of the visual legend. We need to find a place for this in the content flow. It is not interactive, so it may not be relevant at this point. 

However, that said, I will make a stab at it. It will be interesting to know how non-visual users respond to a legend that is mostly for visual understanding.
